options/namesys/opts.go

```diff
@@ -72,3 +72,48 @@ func ProcessOpts(opts []ResolveOpt) ResolveOpts {
 	}
 	return rsopts
 }
+
+// defaultRecordEOL specifies the time that the network will cache IPNS
+// records after being published. Records should be re-published before this
+// interval expires.
+const defaultRecordEOL = 24 * time.Hour
```

Comment:
TODO: 
- make this value follow Expiration (48h) from https://github.com/libp2p/specs/pull/451
  - idealy, use `ProvideValidity` const from  [providers_manager.go#L33-L35](https://github.com/libp2p/go-libp2p-kad-dht/blob/c8b17c2e18f1083dea7742a4f5abb314d9f61ded/providers/providers_manager.go#L33-L35)
- Add paragraph to https://github.com/ipfs/specs/blob/main/ipns/IPNS.md#record-creation suggesting setting Validity to follow  expiration window of publishing medium, and give DHT  expiration interval of 48h as an example. (simple PR, no need for IPIP because we have no default suggestion in specs – we only add info)

Reply:
@lidel https://github.com/ipfs/specs/pull/354